core/src/test/java/nl/nn/adapterframework/jdbc/StoredProcedureQuerySenderTest.java

```diff
@@ -246,6 +276,40 @@ public void testStoredProcedureReturningResultSet() throws Exception {
 		assertXmlEquals(sb.toString(), actual);
 	}
 
+	@Test
+	public void testCallFunction() throws Exception {
+		assumeThat("CALL to custom function only tested on Oracle so far", productKey, equalToIgnoringCase("Oracle"));
+
+		// Arrange
+		sender.setQuery("{ ? = call add_numbers(?, ?) }");
+		sender.setQueryType(JdbcQuerySenderBase.QueryType.OTHER.name());
+		sender.setOutputParameters("1");
+		sender.setScalar(true);
+
+		Parameter dummyParam = new Parameter("dummy", "0");
+		dummyParam.setType(Parameter.ParameterType.INTEGER);
+		dummyParam.configure();
+		sender.addParameter(dummyParam);
+		Parameter p1 = new Parameter("one", "1");
+		p1.setType(Parameter.ParameterType.INTEGER);
+		p1.configure();
+		sender.addParameter(p1);
+		Parameter p2 = new Parameter("two", "2");
+		p2.setType(Parameter.ParameterType.INTEGER);
+		p2.configure();
+		sender.addParameter(p2);
+
+		sender.configure();
+		sender.open();
+
+		// Act
+		SenderResult result = sender.sendMessage(Message.nullMessage(), session);
+
+		// Arrange
+		assertTrue(result.isSuccess());
+		assertEquals("3", result.getResult().asString());
```

Comment:
I suppose the dummyParam will be filled with the output of the function call. Could you also assert on dummyParam value?